src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokemon.java

```diff
@@ -3,51 +3,52 @@
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
+import java.lang.reflect.Type;
 import java.util.List;
 
 public @Data
 class Pokemon {
 
     @SerializedName("id")
-    private int pokemonId;
+    private int id;
     @SerializedName("num")
-    private String pokemonNumber;
+    private int number;
     @SerializedName("name")
-    private String pokemonName;
+    private String name;
     @SerializedName("img")
-    private String pokemonImg;
+    private String img;
     @SerializedName("type")
-    private List<String> pokemonType;
+    private List<String> type;
     @SerializedName("height")
-    private String pokemonHeight;
+    private double height;
     @SerializedName("weight")
-    private String pokemonWeight;
+    private double weight;
     @SerializedName("candy")
-    private String pokemonCandy;
+    private String candy;
     @SerializedName("egg")
-    private String pokemonEgg;
+    private String egg;
     @SerializedName("spawn_choice")
-    private double pokemonSpawnChoice;
+    private double spawnChoice;
     @SerializedName("avg_spawns")
-    private double pokemonAvgSpawns;
+    private double avgSpawns;
     @SerializedName("spawn_time")
-    private String pokemonSpawnTime;
+    private String spawnTime;
     @SerializedName("multipliers")
-    private List<Double> pokemonMultipliers;
+    private List<Double> multipliers;
     @SerializedName("weakness")
-    private List<String> pokemonWeakness;
+    private List<Weakness> weakness;
     @SerializedName("next_evolution")
-    private List<Evolution> pokemonNextEvolution;
+    private List<Evolution> nextEvolution;
     @SerializedName("prev_evolution")
-    private List<Evolution> pokemonPrevEvolution;
+    private List<Evolution> prevEvolution;
```

Comment:
The class is named as Pokemon class. It is clear from the class name that everything is about Pokemon. It is good to remove Pokemon from the variables and name them without using Pokemon.

Reply:
Yes, definitely.